extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import type * as azdataType from 'azdata';
+import * as azdataType from 'azdata';
```

Comment:
removing the `type` from this import will break the sql-database-projects-vscode extension. Use `getAzdataApi()` to be able to call functions like `getServerInfo()`. 

Here's a PR with an explanation of what broke when I accidentally got rid of the `type` in the import a few months ago :) https://github.com/microsoft/azuredatastudio/pull/19292

Reply:
Makes sense. Updated it.